api/db/build.gradle

```diff
@@ -11,16 +11,16 @@ apply plugin: 'org.liquibase.gradle'
 
 def db_host = System.getenv("DB_HOST") ?: "db"
 def db_port = System.getenv("DB_PORT") ?: "3306"
+def liquibase_username = "liquibase"
 def liquibase_password = System.getenv("LIQUIBASE_DB_PASSWORD") ?: "lb-notasecret"
 
 liquibase {
     activities {
         main {
```

Comment:
There's still an issue with this block, at least accorrding to IntelliJ's gradle plugin